src/data-tutorials/users.tsx

```diff
@@ -791,6 +791,13 @@ const Users: User[] = [
       '/blog/tutorials/blockchain-integration-updating-legacy-systems-with-the-celo-platform',
     tags: ['intermediate', 'celosage'],
   },
+  {
+    title: 'How to Connect to a Node on Celo using Docker',
+    description: 'This tutorial teaches how to connect to a node on the celo blockchain using docker',
+    preview: require('./showcase/beginner/https://user-images.githubusercontent.com/69092079/224585198-b09ccdd8-a535-4bad-98f3-b98c382ed714.png'),
```

Comment:
Your image reference here is incorrect
You should reference it from your uploaded header image inside your showcase folder